content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -237,6 +238,73 @@ are available in your cluster, then Pod resource usage can be retrieved either
 from the [Metrics API](/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/#metrics-api)
 directly or from your monitoring tools.
 
+### Considerations for memory backed `emptyDir` volumes {#memory-backed-emptydir}
+
+{{< caution >}}
+If you do not specify a `sizeLimit` for `emptyDir` nor a memory limits
+(`spec.containers[].resources.limits.memory`) for pod, you can create a memory
+backed `emptyDir` with node allocatable size unintentionally. Also, user can
```

Comment:
propose rephrasing:

```
If you do not specify a `sizeLimit` for an `emptyDir` volume, that volume may consume up to that pod's memory limit (`Pod.spec.containers[].resources.limits.memory`).  If you do not set a memory limit, the pod has no upper bound on memory consumption, and can consume all available memory on the node.  Kubernetes schedules pods based on resource requests (`Pod.spec.containers[].resources.requests`) and will not consider memory usage above the request when deciding if another pod can fit on a given node.  This can result in a denial of service and cause the OS to do out-of-memory (OOM) handling.
```

I don't think the aspect of multiple emptyDirs is significant - even one can DoS a machine.

Reply:
Yes! But I'd like to note that having multiple emptyDir increases the risk of OOM significantly. So I'd like to add following:
```
It is possible to create any number of `emptyDir`s that could potentially consume all available memory on the node, making OOM more likely.
```

Reply:
SGTM